dashboard/modules/log/log_agent.py

```diff
@@ -385,8 +385,8 @@ async def StreamLog(self, request, context):
         lines = request.lines if request.lines else 1000
         task_id = request.task_id if request.HasField("task_id") else None
 
-        filepath = f"{self._dashboard_agent.log_dir}/{request.log_file_name}"
-        if not os.path.isfile(filepath):
+        filepath = Path(f"{self._dashboard_agent.log_dir}") / f"{request.log_file_name}"
```

Comment:
```suggestion
        filepath = Path(self._dashboard_agent.log_dir) / request.log_file_name
```

Can you delete a fstring here?